data/mods/chichitech/vitrification/vitrification_effects.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "type": "effect_type",
+    "id": "VITRIFYING",
+    "name": [ "Among Glass" ],
```

Comment:
it isn't. I can't quite figure out why; the Switch function was originally applied because I thought it was part of the same bug that caused multiple u_messages to appear at the same time. The best I can think of for now is that the enchantment applied to use math functions is incapable of applying a steady stat modification by itself. as it stands, I think it's okay; it highlights the sort of half-lucid experience, and as long as it doesn't break something I'm unaware of, having a stat wigging out seems like a pretty good way to ensure you look at it and realize it reflects an intensifying effect over time.